inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/util/LoadNodeUtils.java

```diff
@@ -74,7 +76,10 @@ public static KafkaLoadNode createLoadNode(KafkaSinkResponse kafkaSinkResponse)
                 .map(fieldInfo -> new FieldRelationShip(fieldInfo, fieldInfo)).collect(Collectors.toList());
         Map<String, String> properties = kafkaSinkResponse.getProperties().entrySet().stream()
                 .collect(Collectors.toMap(Map.Entry::getKey, e -> e.getValue().toString()));
-        Integer sinkParallelism = Integer.parseInt(kafkaSinkResponse.getPartitionNum());
+        Integer sinkParallelism = null;
```

Comment:
If sinkParallelism is null, will it have any effect?

Reply:
No effect, sinkParallelism is nullable;